tests/test_lru_cache.py

```diff
@@ -67,3 +72,27 @@ def test_lru_object():
         assert SENTINEL is lru.get(c)
 
     lru.cache_info()
+
+
+def test_expire_object():
+    lru = LruCache(3)
+
+    now_mock = MagicMock()
+    now_mock.return_value = 0
+    lru.now = now_mock
```

Comment:
using `patch` like below would be better.
```python
with mock.patch(lru.now) as p:
    p = 0
    ...
```

For now, it is ok because this is the only test using `LruCache.now`. In future, this may remains side effects to other tests suites.